src/time.rs

```diff
@@ -13,13 +15,8 @@
 /// # Safety
 /// * Causes a GP fault if the TSD flag in register CR4 is set and the CPL
 ///   is greater than 0.
-#[allow(unused_mut)]
-pub unsafe fn rdtsc() -> u64 {
-    let mut low: u32;
-    let mut high: u32;
-
-    asm!("rdtsc" : "={eax}" (low), "={edx}" (high));
-    ((high as u64) << 32) | (low as u64)
+pub unsafe fn rdtsc() -> i64 {
+    _rdtsc()
```

Comment:
Is there a reason why this returns i64 rather than u64?

Reply:
That's how it's defined in [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=rdtsc&expand=4255,4255&othertechs=TSC). However, it seems [C compilers define it as `u64`](https://github.com/gcc-mirror/gcc/blob/f9547530289255c6a4fb6b56c8efb0bcc5e5e1b3/gcc/config/i386/ia32intrin.h#L112). I don't know if this is a bug, or intended behavior. I will raise an issue with `coresimd` and ask there.